_layouts/post.html

```diff
@@ -45,7 +45,7 @@ <h1 data-toc-skip>{{ page.title }}</h1>
       <div class="post-content">
 
         {% if page.image %}
-          <img src="{{ page.image }}" class="preview-img" alt="Preview Image">
+          <img src="{{ page.image }}" class="preview-img" alt="{{ page.image-alt }}" title="{{ page.image-alt }}" />
```

Comment:
1. According to the [MDN docs][title], the `title` attribute should not be the same as `alt`, it is used for tooltips.
2. It would be better to merge the `alt` attribute under the variable `image`, which means:
    ```yml
    ---
    # a post's front matter
    image:
      src: /path/to/img           #  Replacement of original variable `image`
      alt: 'the preview image'
    ---
    ```


[title]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#the_title_attribute